rotkehlchen/chain/evm/decoding/weth/constants.py

```diff
@@ -0,0 +1,58 @@
+import typing
+from dataclasses import dataclass
+from typing import Final
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants.assets import (
+    A_ETH,
+    A_WETH,
+    A_WETH_ARB,
+    A_WETH_BASE,
+    A_WETH_OPT,
+    A_WETH_SCROLL,
+)
+from rotkehlchen.types import SUPPORTED_CHAIN_IDS, ChainID
+
+CPT_WETH: Final = 'weth'
+
+
+@dataclass
+class BaseWrappedETHAsset:
+    eth: Asset
+    weth: Asset
+
+
+CHAINS_WITHOUT_NATIVE_ETH = {
+    ChainID.GNOSIS,
+    ChainID.POLYGON_POS,
+}
+
+
+CHAIN_ID_TO_WETH_MAPPING = {
```

Comment:
This and the others mark them as final

```suggestion
CHAIN_ID_TO_WETH_MAPPING: Final = {
```